google-api-go-generator/testdata/json-body.json

```diff
@@ -451,7 +451,7 @@
                   "type": "string"
                 },
                 "parent": {
-                  "description": "Required. The name of the project for which available locations are to be\nlisted (since some locations might be whitelisted for specific projects).",
+                  "description": "Required. The name of the project for which available locations are to be\nlisted (since some locations might be restricted to for specific projects).",
```

Comment:
```suggestion
                  "description": "Required. The name of the project for which available locations are to be\nlisted (since some locations might be restricted to specific projects).",
```